src/test/java/perosnal/spotifymatcher/service/UserServiceTest.java

```diff
@@ -0,0 +1,131 @@
+package perosnal.spotifymatcher.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import perosnal.spotifymatcher.model.User;
+import perosnal.spotifymatcher.repository.UserRepository;
+import perosnal.spotifymatcher.util.AuthorizedActionResult;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.lenient;
+
+@ExtendWith(MockitoExtension.class)
+class UserServiceTest {
+
+    @Mock
+    private UserRepository userRepository;
+    @Mock
+    private SpotifyApiService spotifyApiService;
+    @InjectMocks
+    private UserService userService;
+
+    @BeforeEach
+     void before() {
+        lenient().when(spotifyApiService.getIdByToken("")).thenReturn("");
+        lenient().when(userRepository.getById("")).thenReturn(new User());
+    }
+
+    @Test
+    void given_short_bio_should_return_failure() {
+        AuthorizedActionResult result = userService.setBio("", "123");
+        assertEquals(result, AuthorizedActionResult.FAILURE);
+    }
+
+    @Test
+    void given_null_bio_should_return_failure() {
+        AuthorizedActionResult result = userService.setBio("", null);
+        assertEquals(result, AuthorizedActionResult.FAILURE);
+    }
+
+    //Biography should be 20 characters or above
+    @Test
+    void given_valid_bio_should_return_success() {
+        AuthorizedActionResult result = userService.setBio("", "123456789Test123456789");
+        assertEquals(result, AuthorizedActionResult.SUCCESS);
+    }
+
+
+
+    @Test
+    void given_already_found_match_should_filter(){
+        User user = validUser();
+        user.getMatches().add(validMatch().getId());
+        List<User> matches = userService.filterMatches(user, new ArrayList<>(Arrays.asList(validMatch())));
```

Comment:
[IDE] Please pay more attention to [your IDE warnings](https://www.jetbrains.com/help/idea/code-inspection.html). This code shows:

> Call to 'asList()' with only one argument 

```suggestion
        List<User> matches = userService.filterMatches(user, new ArrayList<>(Collections.singletonList(validMatch())));
```

ᴛʜɪs ᴀᴄᴛɪᴏɴ ᴡᴀs ᴘᴀʀᴛɪᴀʟʟʏ ᴀᴜᴛᴏᴍᴀᴛᴇᴅ ʙʏ [GɪᴛHᴜʙ's ʀᴇsᴘᴏɴsᴇ ᴛᴇᴍᴘʟᴀᴛᴇs](https://github.com/settings/replies)